compiler/bcq-tools/generate_bcq_metadata

```diff
@@ -190,13 +190,19 @@ def generate_bcq_metadata(flags):
         - Metadata based on each BCQ format
         - Magic number indicating end
     """
-    version = determine_bcq_version(flags)
+    program_version = 1
+    model_version = determine_bcq_version(flags)
 
-    # When there is not BCQ information, do nothing
-    if version == 1:
+    if model_version == 1:
         result_graph_def = generate_bcq_metadata_v1(flags)
-    else:
+    elif model_version == -1:
+        # When there is no BCQ information, do nothing
         result_graph_def = load_graph(flags.input_path)
+    else:
+        err_msg = "BCQ version of the model(v{}) ".format(model_version)
+        err_msg += "is higher than "
+        err_msg += "the version supported by this program(v{})".format(program_version)
+        raise Exception(err_msg)
```

Comment:
This will raise unhandled Exception (which is not accepted by our release testing team).

Please change the code as below.

```suggestion
        raise SystemExit(err_msg)
```